### ISSUE TITLE
implement isHuman Session Flow

### ISSUE THREAD

**Issue by jans510** [dancier/show-dancer#36]
-  Call Backend when user has completed the captcha to initate human session (with cookie)

-  Edit Registration Flow to not send captcha with the POST request of registration (backend will look for "human session cookie instead)

**Comment by jans510** [comment]
-  Call Backend when user has completed the captcha to initate human session (with cookie)

-  Edit Registration Flow to not send captcha with the POST request of registration (backend will look for "human session cookie instead)

**Comment by gorzala** [comment]
I just deployed it and tested it on "production". First it looks good.
1) I open registration
2) I was asked for registration information and the captcha.
3) I send the stuff
4) I got a mail being send.
5) I tried to register again
6) I expected that the system will realize that I have already a human session being established and for that reason to not show the captcha, but it showed the captcha. I would expect (without having looked int he pr) that you only check for the human session before "sutmitting" the form, correct?
